tensorflow-core-ops/Setup.hs

```diff
@@ -90,7 +90,14 @@ fudgePackageDesc lbi p = p
 
 blackList =
     [ -- Requires the "func" type:
```

Comment:
Are all of the below in this category?

Reply:
Yeah, Judah plugged all of the other holes apparently :)